specificatie/kiesrecht.yaml

```diff
@@ -21,18 +21,20 @@ components:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
         einddatumUitsluitingKiesrecht:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
-    Kiesrecht:
+    UitsluitingKiesrecht:
       type: object
       properties:
-        europeesKiesrecht:
-          type: boolean
-          description: |
-            Geeft aan of persoon een oproep moet ontvangen voor verkiezingen voor het Europees parlement.
-          example: true
         uitgeslotenVanKiesrecht:
           type: boolean
           example: true
-        einddatumUitsluitingEuropeesKiesrecht:
-          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
         einddatumUitsluitingKiesrecht:
```

Comment:
hier wordt de groepsnaam "uitsluitingKiesrecht" herhaald in de veldnaam. Is het niet beter dit in te korten naar "einddatumUitsluiting". Het onderscheid met einddatum uitsluiting Europees kiesrecht is immers niet meer nodig (it in andere groep)

Reply:
Aangepast